### DISCUSSION TITLE
Disabling PEP diagnostics errors/warnings

### DISCUSSION THREAD

**Discussion by nevzatseferoglu** [LazyVim/LazyVim#344] [Q&A] [answered]
It is quite annoying seeing those python diagnostic error due to PEP standardization. Is there a way to disable it in LazyVim ?
<img width="601" alt="Screen Shot 2023-02-27 at 16 43 48" src="https://user-images.githubusercontent.com/43863606/221599792-7f33e66a-e8b7-4e79-83fa-a342526a5ef4.png">

**Comment by folke** [comment, marked as answer]
You need to check the lsp server docs on how to disable that.
